Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/dungeon_items_weapons.yml

```
@@ -0,0 +1,418 @@
+    - WeaponCaseShortEnergySwordExpedition
+    - WeaponCaseShortEnergyCutlassExpedition
+    chance: 0.95
+    offset: 0.0
+    rarePrototypes:
+    - SpawnDungeonLootMeleeT5
+    rareChance: 0.01
+
+- type: entity
+  name: random weapon
+  id: SpawnDungeonLootGunT4
+  parent: MarkerBasePlaceFree
+  suffix: "Dungeon, Gun, T4"
+  noSpawn: true
+  components:
+  - type: Sprite
+    layers:
+    - state: green
+      scale: 0.7, 0.7
+    - sprite: Objects/Weapons/Guns/Pistols/mk58.rsi
+      state: icon
+    - sprite: Effects/crayondecals.rsi
+      state: 4
+      color: red
+  - type: RandomSpawner
+    prototypes:
+    - WeaponCaseLongC20r
+    - WeaponCaseLongAkExpedition
+    - WeaponCaseShortSvalinnExpedition
+    - WeaponCaseShortAdvancedLaserExpedition
+    chance: 0.95
+    offset: 0.0
+    rarePrototypes:
+    - SpawnDungeonLootGunT5
+    rareChance: 0.01
+
+- type: entity
+  name: random weapon
+  id: SpawnDungeonLootExplosivesT4
+  parent: MarkerBasePlaceFree
+  suffix: "Dungeon, Explosives, T4"
+  noSpawn: true
+  components:
+  - type: Sprite
+    layers:
+    - state: green
+      scale: 0.7, 0.7
+    - sprite: Objects/Weapons/Bombs/seismic.rsi
+      state: icon
+    - sprite: Effects/crayondecals.rsi
+      state: 4
+      color: red
+  - type: RandomSpawner
+    prototypes:
+    - WeaponCaseShortExGrenade
+    chance: 0.95
+    offset: 0.0
+    rarePrototypes:
+    - SpawnDungeonLootExplosivesT5
+    rareChance: 0.01
+
+# T5: next to unobtainable
+- type: entity
+  name: random weapon
+  id: SpawnDungeonLootMeleeT5
+  parent: MarkerBasePlaceFree
+  suffix: "Dungeon, Melee, T5"
+  noSpawn: true
+  components:
+  - type: Sprite
+    layers:
+    - state: green
+      scale: 0.7, 0.7
+    - sprite: Objects/Weapons/Melee/katana.rsi
+      state: icon
+    - sprite: Effects/crayondecals.rsi
+      state: 5
+      color: red
+  - type: RandomSpawner
+    prototypes:
+    - WeaponCaseShortEnergySwordDoubleExpedition
+    chance: 0.95
+    offset: 0.0
+
+- type: entity
+  name: random weapon
+  id: SpawnDungeonLootGunT5
+  parent: MarkerBasePlaceFree
+  suffix: "Dungeon, Gun, T5"
+  noSpawn: true
+  components:
+  - type: Sprite
+    layers:
+    - state: green
+      scale: 0.7, 0.7
+    - sprite: Objects/Weapons/Guns/Pistols/mk58.rsi
+      state: icon
+    - sprite: Effects/crayondecals.rsi
+      state: 5
+      color: red
+  - type: RandomSpawner
+    prototypes:
+    - WeaponCaseLongBulldog
+    - WeaponCaseLongLauncherRocketExpedition
+    - WeaponCaseLongLauncherChinaLake
+    - WeaponCaseLongLauncherL6
+    chance: 0.95
+    offset: 0.0
+
+- type: entity
+  name: random weapon
+  id: SpawnDungeonLootExplosivesT5
+  parent: MarkerBasePlaceFree
+  suffix: "Dungeon, Explosives, T5"
+  noSpawn: true
+  components:
+  - type: Sprite
+    layers:
+    - state: green
+      scale: 0.7, 0.7
+    - sprite: Objects/Weapons/Bombs/seismic.rsi
+      state: icon
+    - sprite: Effects/crayondecals.rsi
+      state: 4
```

Tier 5, state should be 5.
```suggestion
      state: 5
```